app/org/sagebionetworks/bridge/services/StudyServiceImpl.java

```diff
@@ -220,6 +226,7 @@ public void deleteStudy(String identifier) {
         try {
             lockId = lockDao.acquireLock(Study.class, identifier);
             directoryDao.deleteDirectoryForStudy(identifier);
```

Comment:
change the order of these, if ddb call fails, then don't delete anything else, the study still exists for some reason.